webeoc/src/webeoc-complaints-scheduler/webeoc-complaints-scheduler.service.ts

```diff
@@ -124,13 +113,13 @@ export class WebEOCComplaintsScheduler {
       const response = await axios.post(url, body, config);
       return response.data as Complaint[];
     } catch (error) {
-      this.logger.error('Error fetching complaints from WebEOC:', error);
+      this.logger.error("Error fetching complaints from WebEOC:", error);
       throw error;
     }
   }
 
   // Formats a date to be in a format required by WEBEOC.  Used to determine how far back in time to go to grab new complaints.
   private formatDate(date: Date): string {
-    return date.toISOString().replace('T', ' ').substring(0, 19);
+    return date.toISOString().replace("T", " ").substring(0, 19);
```

Comment:
This doesn't seem like a good idea here. A better solution would be to use datefns to format your date. It shouldn't happen but this feels like it has the potential to explode and because its happening outside of user control the chances of catching it could be difficult.

Reply:
Good find, I'll make that change.

Reply:
Fixed